package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "global-tunnel-ng",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
Can you please remove the version push? This is already part of the automated release procedure, thus we would jump one empty version.